media/css/firefox/new2.less

```diff
@@ -0,0 +1,579 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+@import "../sandstone/lib.less";
+
+.sky #outer-wrapper {
+    background: #eaeff2;
+    background: -webkit-gradient(radial, center center, 0px, center center, 60%, color-stop(0%,rgba(234,239,242,.9)), color-stop(60%,rgba(212,221,228,.5))),
```

Comment:
I think we can get rid of this one too? Only the most recent -webkit prefixed version of the standard property should be needed still.